tests/integration/test_alembic_cp01.py

```
@@ -0,0 +1,70 @@
+
+    resolved = shutil.which("alembic")
+    if resolved is None:
+        pytest.fail("alembic executable is not available in the active environment")
+    return [resolved]
+
+
+def _run_alembic(*args: str) -> subprocess.CompletedProcess[str]:
+    assert PMS_TEST_DATABASE_URL is not None
+    env = os.environ.copy()
+    env["DATABASE_URL"] = PMS_TEST_DATABASE_URL
+    return subprocess.run(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)